libs/search-widget/src/widgets/search-widget/Widget.svelte

```diff
@@ -110,6 +111,7 @@
           activateEditLabelsFeature();
         }
       }),
+      canAnnotateEntities.subscribe(canAnnotate => entityAnnotationFeatureEnabled.set(canAnnotate))
```

Comment:
one question, if canAnnotateEntities and entityAnnotationFeatureEnabled have the same value, do we need both to exist ?

Reply:
you're right, we need only one.